runners/core-java/pom.xml

```diff
@@ -95,6 +95,11 @@
       <artifactId>joda-time</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
```

Comment:
I think this should live in its own module that is opt-in separate from runners-core.